tests/lang_callable/macros/tif_ill_formed.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+  cmd: "nim check $options $file"
+  action: reject
+  nimout: '''
+tif_ill_formed.nim(13, 15) Error: illformed AST:  else: <<0th child missing for nkElseExpr >>
+'''
```

Comment:
Could you use inline annotations (`#[tt.Error`) instead? They make it easier to extend this test in the future, as one doesn't have to update all line numbers.